src/main/java/bowling/model/BowlingGame.java

```diff
@@ -0,0 +1,64 @@
+package bowling.model;
+
+import java.util.List;
+
+public class BowlingGame {
+
+  public final static int MAX_NUMBER_OF_FRAMES = 10;
+
+  private Frames frames = new Frames();
+  private int bonusCnt;
```

Comment:
불필요한 약어를 사용하지 말고, 끝까지 써주세요.
```suggestion
    private int bonusCount;

Reply:
4c3a30f
수정했습니다.